mama/jni/src/com/wombat/mama/MamaSubscription.java

```diff
@@ -81,23 +81,23 @@ public MamaSubscription()
      * @throws Throwable A MamaException will be thrown if the underlying
      *                   subscription could not be de-allocated.
      */
-    protected void finalize() throws Throwable
-    {
-        try 
-        {
-            // Only deallocate the subscription if the native pointer is valid
-            if(subscriptionPointer_i != 0)
-            {
-                deallocate();
-            }
-        }
-
-        finally
-        {
-            // Call the base class finalizer
-            super.finalize();
-        }
-    }
+    // protected void finalize() throws Throwable
```

Comment:
Done - will resubmit this as having a finalizer in java does not do anything since to get to finalize you have to call destroy first to clear the references, and destroy does all that finalize does.